content/_data/changelogs/weekly.yml

```diff
@@ -576,8 +576,50 @@
         Internal: Offer default methods on <code>ParameterizedJob</code> to have less boilerplate code.
       pull: 2864
   # pull 2884: too minor (Javadoc)
-
-
+- version: "2.62"
+  date: 2017-05-21
+  changes:
+    - type: rfe
+      message: >
+        Fixed Pipeline compatibility for a number of CLI commands (<tt>delete-builds</tt>, <tt>list-changes</tt>, <tt>console</tt>, <tt>set-build-description</tt>, and <tt>set-build-display-name</tt>),
+        as well as some issues affecting error reporting in other commands when used with Pipeline.
+      pull: 2874
+      references:
+        - issue: 30785
+        - issue: 41527
+    - type: rfe
+      message: >
+        If you have the Authorize Project plugin installed and configured, its configuration will now be treated as final with respect to the behavior of <em>Job/Build</em> checks from <strong>Build other projects</strong> and <strong>Build after other projects are built</strong>.
```

Comment:
Installation numbers from https://plugins.jenkins.io/authorize-project don't justify this. Also, not a huge change in behavior AFAIUI.